descriptors/ModuleDescriptor-template.json

```diff
@@ -222,26 +222,10 @@
       "id" : "login",
       "version" : "6.0 7.0"
     },
-    {
-      "id" : "service-points",
-      "version" : "2.1 3.0"
-    },
-    {
-      "id" : "service-points-users",
-      "version" : "1.0"
-    },
-    {
-      "id": "password-validator",
-      "version": "1.0"
-    },
```

Comment:
What is the behaviour of change password when the password validation interface is not provided?

Reply:
A brief exploration of the current mainline code suggests that any non-200 status code response is considered a validation failure and that results in a 500 error for change password and (I think) an unhandled failure during password reset.

This suggests to me that neither of these processes can fulfil their intended purpose without this interface being present.